source/docs/index.md

```diff
@@ -1,102 +1,22 @@
-title: Documentation
+title: Getting Started
 ---
-Welcome to the Embark documentation. If you encounter any problems when using Embark, have a look at the  [troubleshooting guide](troubleshooting.html), raise an issue on [GitHub](https://github.com/embark-framework/embark/issues) or ask in the [Gitter Chatroom](https://gitter.im/embark-framework/Lobby).
+Welcome to the Embark documentation! The following guides will help you guiding your way through your journey of building decentralized applications. If you encounter any problems when using Embark, or find anything unclear described here, please have a look at the [troubleshooting guide](troubleshooting.html), raise an issue on [GitHub](https://github.com/embark-framework/embark/issues) or ask in the [Gitter Chatroom](https://gitter.im/embark-framework/Lobby) respectively.
 
 ## What is Embark?
 
-Embark is a fast, simple and powerful framework to develop and deploy Decentralized Applications (DApps). 
+Embark is a fast, easy to use, and powerful developer environment to build and deploy decentralized applications, also known as "DApps". It integrates with [Ethereum](https://ethereum.org/) blockchains, decentralized storages like [IPFS](https://ipfs.io) and [Swarm](https://swarm-guide.readthedocs.io/en/latest/introduction.html), and decentralized communication platforms like [Whisper](https://github.com/ethereum/wiki/wiki/Whisper) and [Orbit](https://github.com/orbitdb/orbit).
 
-Embark currently integrates with EVM blockchains (Ethereum), Decentralized Storages (IPFS), and Decentralized communication platforms (Whisper and Orbit). Swarm is supported for deployment.
+Embark's goal is to make building decentralized applications as easy as possible, by providing all the tools needed and staying extensible at the same time.
 
-With Embark you can:
+Some of Embark's features, but not all of them, are:
 
-**Blockchain (Ethereum)**
-* Automatically deploy contracts and make them available in your JS code. Embark watches for changes, and if you update a contract, Embark will automatically redeploy the contracts (if needed) and the dapp.
-* Contracts are available in JS with Promises.
-* Do Test Driven Development with Contracts using Javascript.
-* Keep track of deployed contracts; deploy only when truly needed.
-* Manage different chains (e.g testnet, private net, livenet)
-* Easily manage complex systems of interdependent contracts.
+* **Automatic Smart Contract deployment** - Embark takes care of deploying your Smart Contracts as well as redeploying them as you make changes to your code.
+* **Client development** - Smart Contracts are not a must! Build your application with the framework of your choice right within Embark.
```

Comment:
Not sure what you mean by that? Smart contracts are pretty important, so not sure if you meant Solidity is not a must

Reply:
Oh sorry, yea this probably needs to be reworded. And also, I think I had a brain twist here.

The point was that you can use Embark and decentralization features without actually making your app dependent on Smart Contracts. E.g. if you just want to build a SPA that's deployed via IPFS, you could do that. 

But maybe we want to turn this around and focus on the fact that you can just deploy Smart Contracts without building a SPA?

Reply:
Ah ok, I understand. Maybe just leave the smart contract part out and just talk about Front end development using the framework of your choice.